tests/Testing/TestResponseTest.php

```diff
@@ -564,6 +564,22 @@ public function testAssertUnauthorized()
         $response->assertUnauthorized();
     }
 
+    public function testAssertBadRequest()
+    {
+        $statusCode = 500;
```

Comment:
Shouldn't this be 400 to match HTTP status `400 - bad request`?

Reply:
for this tests no, check the other `400` methods.

Reply:
> for this tests no, check the other `400` methods.

Ah! Didn't noticed it. Do you know why it should be `500` and not `400`? I think it is confusing..